src/gazebo2rviz/gazebo2moveit_collision_node.py

```diff
@@ -18,6 +20,9 @@
 updatePeriod = 0.5
 model_cache = {}
 sdf2moveit = None
+# Service allowing to map a named object to its type
```

Comment:
unnecessary comment. If the name is still unclear, please just try updating it so it clearly reflects what it is doing

Reply:
Btw, this is more a suggestion and consideration rather then a critique, according to personal coding philosophy ;)